src/Parser/Rogue/Subtlety/Modules/ResourceTracker/ResourceTracker.js

```diff
@@ -74,7 +74,7 @@ class ResourceTracker extends Analyzer {
     
     if (!(spellId in this.spendersObj)) {
       this.initSpenderAbility(spellId);
-  }
+    }
 
     const eventResource = this.getResource(event);
 
```

Comment:
you should include this, as said in the original review, to ensure no crashes (i've been bugfixing tons in my modules because of this, despite thinking stuff isn't possible, it will inevitably happen, might aswell be safe) 
```jsx 
if (!cost) { 
      return; 
}